terminal/src/io.rs

```diff
@@ -3,9 +3,7 @@ use std::str::FromStr;
 
 use zeroize::Zeroizing;
 
-use librad::crypto::keystore::pinentry::SecUtf8;
-use librad::crypto::BoxedSigner;
-use librad::profile::Profile;
+use secstr::SecUtf8;
```

Comment:
previously, we were using `librad::crypto::keystore::pinentry::SecUtf8;` but using `secstr::SecUtf8` compiles just fine. hope this didn't break anything.